apps/trading/client-pages/markets/market-icon.tsx

```diff
@@ -0,0 +1,72 @@
+import { type MarketMaybeWithData } from '@vegaprotocol/markets';
+import { MarketTradingMode, MarketState } from '@vegaprotocol/types';
+import { VegaIconNames, VegaIcon, Tooltip } from '@vegaprotocol/ui-toolkit';
+import { useT } from '../../lib/use-t';
+
+export const getTradingModeIcon = (
+  tradingMode: MarketTradingMode,
+  state?: MarketState
+): VegaIconNames | null => {
+  switch (tradingMode) {
+    case MarketTradingMode.TRADING_MODE_MONITORING_AUCTION:
+    case MarketTradingMode.TRADING_MODE_OPENING_AUCTION:
+    case MarketTradingMode.TRADING_MODE_BATCH_AUCTION:
+      return VegaIconNames.HAMMER;
+    case MarketTradingMode.TRADING_MODE_SUSPENDED_VIA_GOVERNANCE:
+      return VegaIconNames.PAUSE;
+    case MarketTradingMode.TRADING_MODE_NO_TRADING:
+      switch (state) {
+        case MarketState.STATE_PENDING:
+        case MarketState.STATE_PROPOSED:
+          return VegaIconNames.MONITOR;
+        default:
+          return VegaIconNames.CLOSED;
+      }
+    default:
+      return null;
+  }
+};
+
+export const getTradingModeTooltip = (
+  tradingMode?: MarketTradingMode,
+  state?: MarketState
+): string => {
+  switch (tradingMode) {
+    case MarketTradingMode.TRADING_MODE_MONITORING_AUCTION:
+      return 'In protective auction due to large price movement, crossed orders are required to set price and restart trading';
```

Comment:
**chore (blocking)**:  These should be i18n strings

Reply:
yes, they are translated inside the component. 
that is because we need the `useT` hook that can not be initialised here.